### DISCUSSION TITLE
parameters inside the query

### DISCUSSION THREAD

**Discussion by ToniTaste** [sqlitebrowser/sqlitebrowser#2919] [Q&A] [answered]
Hi,

the query `SELECT * FROM car WHERE color = :param`  doesn't show a parameter window but also no error. How can I make queries with parameters? 

Toni

**Comment by chrisjlocke** [comment]
I don't think using straight SQL (and in DB4S) you can use parameters.  SQLite supports it, but only programmatically.  I might be wrong though...
So while the statement you entered is correct, you also have to tell SQLite what the parameters are.  You can't do this in the GUI.

**Comment by chrisjlocke** [comment, marked as answer]
Reading StackOverflow, you can use the sqlite3 command-line tool, which does support parameters.
https://stackoverflow.com/questions/56640238/sqlite-can-we-pass-arguments-to-a-sql-file-from-batch-file

but again, not from within DB4S.
